app/accounts/views.py

```
@@ -19,6 +21,15 @@ class InviteConsumeFormView(UUIDSlugMixin, SingleObjectMixin, FormView):
     context_object_name = "invite"
     success_url = "/"
 
+    def dispatch(self, request, *args, **kwargs):
+        try:
+            kwargs["slug"] = SmallUUID(kwargs["slug"]).hex_grouped
+            self.kwargs["slug"] = kwargs["slug"]
+            statsd.increment("turnout.accounts.invite_smalluuid_key_usage")
+        except Exception:
```

Can you be more specific about this Exception? Like is this supposed to fail if `kwargs["slug"]` isn't a SmallUUID? I'm worried that this will end up swallowing errors from, e.g., `statsd.increment`.

---

Done